### ISSUE TITLE
API: Integrate the download class data

### ISSUE THREAD

**Issue by Chaos10001** [gray-adeyi/orderlie-fe-react#61]
Check endpoints here: https://orderlies-555f0367f3ea.herokuapp.com/docs#/

Endpoint: /api/v1/{department_id}/classes/{class_id}/download

**Comment by 0xlarmideh** [comment]
@Chaos10001 Please assign this to me

**Comment by devchaos0** [comment]
@0xlarmideh  You can Work on this, Once you done with this i will assign you to the rest you asked for.

**Comment by 0xlarmideh** [comment]
Awesome

**Comment by 0xlarmideh** [comment]
@Chaos10001 Where is this to be integrated?

**Comment by devchaos0** [comment]
Check the figma link for the download class data

**Comment by 0xlarmideh** [comment]
That means I will be building out the UI also @Chaos10001 ?

**Comment by devchaos0** [comment]
@0xlarmideh the UI is already built try to git clone the orderlie redo...

**Comment by 0xlarmideh** [comment]
@Chaos10001 I've done this but I need a clarification. Currently, the payload to the POST request is not available.... Should I just write the function?

**Comment by 0xlarmideh** [comment]
Also...I added a dummy payload and it returns strict-origin-when-cross-origin error. You might want to check that also.

**Comment by devchaos0** [comment]
Check endpoints here: https://orderlies-555f0367f3ea.herokuapp.com/docs#/

Endpoint: /api/v1/{department_id}/classes/{class_id}/download

Use this endpoint chief

**Comment by 0xlarmideh** [comment]
@Chaos10001 That's the endpoint I'm using.... department_id, class_id...are those not variables being passed to that api route slug?

Where are those coming from?

**Comment by devchaos0** [comment]
Check the API docs chief...

The endpoint I wrote is just to navigate you over there...

Once you implement the API and you see that it works, then just make a push, the maintainer will check it out

**Comment by 0xlarmideh** [comment]
Yeah, sure. From my understanding and what i've done everywhere,  those variables in labels in curly brackets are variables that need to be defined and passed to that api slug. Example department_id: 'd563737vdbbd' class_id: '574brbfbhb3883'.

Without those being defined, I'm sorry, but it won't work. 

However, if I'm getting this wrong, you can show me an example.

**Comment by devchaos0** [comment]
Oh... The backend guys will get back to you on this then
